<launch>	
	<rosparam command="load" file="$(find golfcar_odom)/launch/low_pass.yaml"/>
	<node pkg="golfcar_odom" type="speedFilter" name="speedFilter"/>
</launch>
